packages/ui/src/working-groups/components/WorkingGroupImage.tsx

```diff
@@ -257,6 +257,39 @@ export const WorkingGroupImage = ({ className, groupName }: WorkingGroupImagePro
           />
         </WorkingGroupImageTag>
       )
+    case 'council':
+      return (
+        <WorkingGroupImageTag viewBox="0 0 108 108" fill="none" className={className}>
+          <path
+            d="M61.3984 31.1091C61.3984 33.5871 60.1671 35.7791 58.2799 37.1126C60.2961 37.8978 61.7903 39.2235 62.8425 40.7469C64.4716 43.1057 65 45.8622 65 47.6673V48.7445L64.0314 49.2262C62.5639 49.9561 58.7207 51.25 54 51.25C49.2793 51.25 45.4361 49.9561 43.9686 49.2262L43 48.7445V47.6673C43 45.8622 43.5284 43.1057 45.1575 40.7469C46.2097 39.2234 47.704 37.8977 49.7203 37.1125C47.8332 35.779 46.602 33.5871 46.602 31.1091C46.602 27.0448 49.9143 23.75 54.0002 23.75C58.0861 23.75 61.3984 27.0448 61.3984 31.1091Z"
+            fill={Colors.Blue[500]}
+          />
+          <path
+            d="M39.3984 44.8591C39.3984 47.3371 38.1671 49.5291 36.2799 50.8626C38.2961 51.6478 39.7903 52.9735 40.8425 54.4969C42.4716 56.8557 43 59.6122 43 61.4173V62.4945L42.0314 62.9762C40.5639 63.7061 36.7207 65 32 65C27.2793 65 23.4361 63.7061 21.9686 62.9762L21 62.4945V61.4173C21 59.6122 21.5284 56.8557 23.1575 54.4969C24.2097 52.9734 25.704 51.6477 27.7203 50.8625C25.8332 49.529 24.602 47.3371 24.602 44.8591C24.602 40.7948 27.9143 37.5 32.0002 37.5C36.0861 37.5 39.3984 40.7948 39.3984 44.8591Z"
+            fill={Colors.Blue[500]}
+          />
+          <path
+            d="M83.3984 44.8591C83.3984 47.3371 82.1671 49.5291 80.2799 50.8626C82.2961 51.6478 83.7903 52.9735 84.8425 54.4969C86.4716 56.8557 87 59.6122 87 61.4173V62.4945L86.0314 62.9762C84.5639 63.7061 80.7207 65 76 65C71.2793 65 67.4361 63.7061 65.9686 62.9762L65 62.4945V61.4173C65 59.6122 65.5284 56.8557 67.1575 54.4969C68.2097 52.9734 69.704 51.6477 71.7203 50.8625C69.8332 49.529 68.602 47.3371 68.602 44.8591C68.602 40.7948 71.9143 37.5 76.0002 37.5C80.0861 37.5 83.3984 40.7948 83.3984 44.8591Z"
+            fill={Colors.Blue[500]}
+          />
+          <path
+            d="M40.3596 54.2417L42.7155 52.5249H65.2845L67.6405 54.2417L75.606 78.9917L70.894 80.5082C56.9093 83.3147 42.5417 81.6776 37.106 80.5082L32.394 78.9917L40.3596 54.2417Z"
+            fill={Colors.Black[500]}
+          />
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M70.9472 64.5162C68.6333 64.073 66.8614 63.4202 65.9686 62.9762L65 62.4945V61.4173C65 59.6122 65.5284 56.8557 67.1575 54.4969C67.2501 54.3628 67.3461 54.2304 67.4455 54.0996L67.6405 54.2417L70.9472 64.5162Z"
+            fill={Colors.Black[900]}
+          />
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M40.5545 54.0996C40.654 54.2304 40.7499 54.3628 40.8425 54.4969C42.4716 56.8557 43 59.6121 43 61.4173V62.4945L42.0314 62.9762C41.1387 63.4202 39.3667 64.073 37.0529 64.5162L40.3596 54.2417L40.5545 54.0996Z"
+            fill={Colors.Black[900]}
+          />
+        </WorkingGroupImageTag>
+      )
     case 'gateways':
```

Comment:
```suggestion
      )
    case 'gateway':
```

Ohhhh, that's why :D

Reply:
![](https://user-images.githubusercontent.com/247363/128518907-ccc23400-ee09-4a9d-9648-516a33fc3ea9.png)

When switching to "Olympia-testnet" in the settings.